app/src/app/components/specmath-modal/modal.component.ts

```diff
@@ -147,8 +148,8 @@ export class ModalComponent {
   }
 
   async mergeOperation() {
-    // ?Replace the mock service with real once its deployed
-    const callResponse = await this.mockService.mergeSpecsConflicts().toPromise();
+    const mergeSet = await this.generateMergeSet();
+    const callResponse = await this.mockService.mergeSpecs(mergeSet).toPromise();
```

Comment:
Missed this before but since this could be the real service, name it as if it were real.

Reply:
Updated 👍